### PR TITLE
Nuclear bombs now actually detonate

### PR DESCRIPTION
**Pull request by Cyberboss**
:cl: Cyberboss
experiment: Nuclear bombs now detonate
/:cl:

Reasoning: Let the people have their round end cake

Won't affect current objective completion, anyone on the zlevel with the bomb is considered dead once it goes off

I think this also fixes a bug where you can avoid watching the explosion animation

### CODE REVIEW

**Comment by Cyberboss** [comment]
Will reopen after the freeze

**Comment by francinum** [comment]
do you want lag,
that's how you get lag.

**Comment by Cyberboss** [comment]
Just let it happen ( ͡° ͜ʖ ͡°)

**Comment by jughu** [comment]
Inb4 people complain round end takes longer

**Comment by Shapsy** [comment]
@francium this is actually fairly lagless, admins have dropped 50/100/200 bombs without halting the server for months now thanks to mso

File: code/modules/mob/mob_defines.dm [hunk 1]
```diff
@@ -145,3 +145,4 @@
 	var/resize = 1 //Badminnery resize
 
 	var/list/observers = null	//The list of people observing this mob.
+	var/nuked = FALSE	//prevent "hurr-durr i survived" after a nuke goes off
```
Comment (RemieRichards) [inline code/modules/mob/mob_defines.dm]: No, just kill them.

File: code/controllers/subsystem/ticker.dm [hunk 2]
```diff
@@ -207,31 +207,38 @@ var/datum/subsystem/ticker/ticker
 
 	return 1
 
+/datum/subsystem/ticker/proc/station_explosion_detonation(atom/bomb)
+	if(bomb)	//BOOM
+		var/turf/epi = bomb.loc
+		qdel(bomb)
+		explosion(epi, 255, 255, 255, 255, TRUE, TRUE, 0, TRUE)
```
Comment (ChangelingRain) [inline code/controllers/subsystem/ticker.dm]: don't do maximum size, make it leave a very... battered shell of a station

Comment (Cyberboss) [inline code/controllers/subsystem/ticker.dm]: That would actually be MORE laggy. Plus it doesn't even get halfway done before banging donk

**Comment by Cyberboss** [comment]
MSO said so

**Comment by KorPhaeron** [comment]
PR doesn't need to be open to be test merged

File: code/controllers/subsystem/ticker.dm (hunk 2 above)
Comment (MrStonedOne) [inline code/controllers/subsystem/ticker.dm]: Make it do explosion(epi, 0, 256, 512, 0, TRUE, TRUE, 0, TRUE).

I've tested, it looks pretty

**Comment by Cyberboss** [comment]
This ain't a fix :/

**Comment by Iamgoofball** [comment]
sshh

**Comment by calzilla1** [comment]
IM NUCLEAR, I'M WILD, IM BRREAKIMG UP INSIIIIIDE

**Comment by JJRcop** [comment]
what's up with this?

**Comment by Cyberboss** [comment]
It's been ready forever

**Comment by kevinz000** [comment]
CYBERBOSS FIX YOUR SHIT REE

**Comment by Cyberboss** [comment]
fuck wat happen

**Comment by kevinz000** [comment]
Kor nuked bagil and said something about nukes not working right.

**Comment by blobbernaut** [comment]
"thanks" to MSO

new bombs are buggy pieces of fucking garbage but sure

**Comment by francinum** [comment]
>Jan 30th

**Pull request merged**